### PR TITLE
[ODS-6574] Documentation for SerializedData and ResourceLinks features

### PR DESCRIPTION
**Pull request by gmcelhanon**

### CODE REVIEW

File: odsApi_versioned_docs/version-7.2/platform-dev-guide/configuration/readme.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ instance of the Ed-Fi ODS / API. By customizing the `appsettings.json` you
 should be able to modify the API to suit your needs. `appsettings.json` includes
 settings for database connection strings and the settings that enable various
 configurable features. For more detail on the configuration files and the
-specific values, see the [Configuration Details](./configuration-details.md)
+specific values, see the [Configuration Details](./configuration-details.mdx)
```
Comment (vimayya) [inline odsApi_versioned_docs/version-7.2/platform-dev-guide/configuration/readme.md:14]: can you revert this change. As 7.2 space still has the md file, build is reporting broken link

File: odsApi_versioned_docs/version-7.3/platform-dev-guide/configuration/configuration-details.mdx [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+Examples of required configurations include database connection strings, SMTP
+server locations, and similar.
+
+To make it easier for developers to install and run the Ed-Fi ODS / API, the
+default download from source control is pre-configured with values appropriate
+for a developer or single-server test instance of the system.
+
+### EdFi.Ods.SwaggerUI
+
+In _appsettings.json_:
```
Comment (vimayya) [inline odsApi_versioned_docs/version-7.3/platform-dev-guide/configuration/configuration-details.mdx:28]: Formatting looks much better now. I wonder if we should just write at the top section something along the lines of - _In Ed-Fi ODS / API, appsettings.json provides the primary configuration for the ASP.NET Core applications._ - and remove this at the beginning of each section.

File: odsApi_versioned_docs/version-7.2/platform-dev-guide/configuration/readme.md (hunk 1 above)
Comment (gmcelhanon) [reply odsApi_versioned_docs/version-7.2/platform-dev-guide/configuration/readme.md]: Done.
